<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="MUMTEWD0500" name="Command line test 2a5faf9d-ce05-4fa6-939a-7f6ebe23054f" tests="3" failures="1" timestamp="23 Mar 2018 07:05:42 GMT" time="23.937" errors="0">
  <testcase name="openBrowser" time="9.827" classname="com.HReckAutomation.HReck.HReckLogin"/>
  <testcase name="hReckLogin" time="14.038" classname="com.HReckAutomation.HReck.HReckLogin">
    <failure type="java.lang.AssertionError" message="Test case passed expected [Enter valid password] but found [http://hreck.techeasesystems.in/#/login]">
      <![CDATA[java.lang.AssertionError: Test case passed expected [Enter valid password] but found [http://hreck.techeasesystems.in/#/login]
	at com.HReckAutomation.HReck.HReckLogin.hReckLogin(HReckLogin.java:58)
	at com.mentorstudies.automationframework.TestRunner.main(TestRunner.java:117)
	at org.codehaus.mojo.exec.ExecJavaMojo$1.run(ExecJavaMojo.java:294)
	at java.lang.Thread.run(Thread.java:748)
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- hReckLogin -->
  <testcase name="hReckLogin" time="0.057" classname="com.HReckAutomation.HReck.HReckLogin"/>
</testsuite> <!-- Command line test 2a5faf9d-ce05-4fa6-939a-7f6ebe23054f -->
